src/components/navigation/Navbar.jsx

```diff
@@ -0,0 +1,54 @@
+import { BsMoonFill, BsPlayCircleFill, BsSun } from "react-icons/bs";
+import { useEffect } from "react";
+import styles from "./navigation.module.css";
+import { Link } from "react-router-dom";
+import { useLocalStorage } from "../../utils/hooks/useLocalStorage";
+
```

Comment:
you can add a jsconfig.json file to your global folder, where your env file is basically. There in the baseurl if you include src folder in compiler options then you can avoid 
.. /..  Everywhere

Reply:
This is a new learning for me, I will surely incorporate in the project. Thanks for sharing.